drucker_dashboard/server.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+
+import os
+
+from flask import Flask, render_template
+from flask_cors import CORS
+
+try:
+    from my_dashboard_logger import logger
+except ImportError:
+    from drucker_dashboard.logger import logger
+
+from drucker_dashboard.utils import DruckerDashboardConfig
+from drucker_dashboard.models import db
+from drucker_dashboard.apis import api
+from drucker_dashboard.auth import auth
+
+
+def create_app(config_file: str = "settings.yml"):
+    """create_app."""
+
+    app = Flask(__name__, static_folder='static', template_folder='static/dist')
+
+    @app.route('/')
+    @app.route('/login')
+    @app.route('/settings/kubernetes/hosts')
+    @app.route('/settings/kubernetes/hosts/add')
+    @app.route('/settings/kubernetes/hosts/<int:kubernetes_id>/edit')
+    @app.route('/applications')
+    @app.route('/applications/add')
+    @app.route('/applications/<int:application_id>/dashboard')
+    @app.route('/applications/<int:application_id>/services')
+    @app.route('/applications/<int:application_id>/services/add')
+    @app.route('/applications/<int:application_id>/services/<int:service_id>/edit')
+    @app.route('/applications/<int:application_id>/models')
+    @app.route('/applications/<int:application_id>/models/<int:model_id>/edit')
+    @app.route('/applications/<int:application_id>/admin')
+    def root_url(**kwargs):
+        return render_template('index.html')
```

Comment:
It might be forgettable to add a new route here every time a new page is added to frontend.

So what about changing to like this?
```
def create_app(config_file: str = "settings.yml"):
    app = Flask(__name__, static_folder='static')

    @app.route('/')
    @app.route('/<path:path>') # this is wildcard routing
    def root_url(**kwargs):
        print(kwargs)
        return render_template('index.html')
```

API endpoints has higher priority than `@app.route('/<path:path>')` , so wildcard endpoint does not affect API endponts

Reply:
Sounds good.
I need to minor modification to escape `/doc` entrypoint for swagger UI.
See https://github.com/rekcurd/drucker-dashboard/pull/54/commits/47aa0c9d5da99f136c1959adaccef6a12a66b999

Reply:
Actually I confirmed `@app.route('/<path:path>')` does not affect `/doc` , either. (Last time I checked it out, I checked `/docs` by mistake...)

Reply:
I have checked also and have confirmed it works fine.
Thank you!